_posts/2024-03-19-WildFly-mini-conference-recap.adoc

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "WildFly Mini Conference"
+date:   2024-03-19
+tags:   conference
+author: hpehl
+description: WildFly Mini Conference
+---
+
+= WildFly Mini Conference
+
+Many thanks to all participants of the first WildFly Mini Conference!
+
+Our first WildFly Mini Conference took place on March 6. We had four talks on the https://www.wildfly.org/conference/[agenda] and many interesting questions and feedback. The feedback we have received so far has been very positive and has encouraged us to organize similar events for the community in the future. Stay tuned!
+
+If you haven't done so far, please let us know how you liked the conference using our https://forms.gle/ruEytViEBBwthr5K7[feedback form].
```

Comment:
Maybe after this sentence, we could mention that if anyone missed the conference, they can watch the recording?

Reply:
I added a sentence about that.